spec/acts_as_spec.rb

```diff
@@ -223,6 +223,14 @@
       expect(pen.updated_at).not_to eq(update)
     end
 
+    it "touches supermodel only when attributes changed" do
+      pen.save
+      pen.reload
+      update = pen.product.updated_at
```

Comment:
You could write `expect { pen.save }.to_not change { pen.reload.updated_at }`, it reads nicer IMO 😄